source/configuration_manual/authentication/auth_policy.rst

```diff
@@ -168,6 +170,48 @@ produces
 
 for this to work.
 
+List of fields
+--------------
+
+All fields supported by :ref:`variables-auth` can be used. In addition, you can use following fields:
+
+``hashed_password``
+  User's password hashed with :ref:`authentication-policy_password_hash_algorithm`.
+
+``requested_username``
+  Username that was requested. This is a field which contains the correct username,
+  depending on whether this is master user login or not.
```

Comment:
Still a bit unclear what is "the correct username". I guess the intention is something like: `Username for regular logins. For master user logins, this is the requested login username (not the master username).`?